.github/actions/nm-set-env/action.yml

```diff
@@ -9,7 +9,7 @@ runs:
   steps:
   - run: |
       echo "HF_HOME=${HF_HOME_TOKEN}" >> $GITHUB_ENV
-      echo "TORCH_CUDA_ARCH_LIST=8.0+PTX" >> $GITHUB_ENV
+      echo "TORCH_CUDA_ARCH_LIST=7.0 7.5 8.0 8.6 8.9+PTX" >> $GITHUB_ENV
```

Comment:
is there any reason we aren't using what is used in the existing vllm build system? IMO this should be handled by https://github.com/neuralmagic/neuralmagic-vllm/blob/a3f00c562cd8f2db17d8f7a61b1de0210b0b2275/setup.py#L21

Reply:
we aren't really using there build system